subprojects/tooling-api/build.gradle.kts

```diff
@@ -60,6 +61,7 @@ dependencies {
     testImplementation(testFixtures(project(":dependency-management")))
     testImplementation(testFixtures(project(":ide")))
     testImplementation(testFixtures(project(":workers")))
+    testImplementation(testFixtures(project(":toolchains-jvm")))
```

Comment:
❓ Are these two new dependencies needed? It seems to me they are not.

Reply:
You are right. I should have removed this when we decided that cross-version tests, due to their limitations, can't use the fixture I've added.